src/providers/node/mod.rs

```diff
@@ -160,23 +180,49 @@ impl NodeProvider {
                 if start_target.configurations.is_some()
                     && start_target.configurations.unwrap().production.is_some()
                 {
-                    return Ok(Some(format!("npx nx run {}:start:production ", app_name)));
+                    return Ok(Some(format!(
+                        "{} nx run {}:start:production ",
+                        dlx, app_name
+                    )));
                 }
-                return Ok(Some(format!("npx nx run {}:start", app_name)));
+                return Ok(Some(format!("{} nx run {}:start", dlx, app_name)));
             }
 
             if project_json.targets.build.executor == "@nrwl/next:build" {
-                return Ok(Some(format!("cd {} && npm run start", output_path)));
+                return Ok(Some(format!(
+                    "cd {} && {} run start",
+                    pkg_manager, output_path
+                )));
             }
 
             let main = project_json.targets.build.options.main;
+            let executor = if pkg_manager == "bun".to_string() {
```

Comment:
nit: Use the `executor` everywhere else that `node` is used for the start command (not just for NX apps).